monitoring/rid_qualifier/flight_data_generator.py

```diff
@@ -196,7 +200,17 @@ def utm_converter(self, shapely_shape: shapely.geometry, inverse:bool=False) ->
         
         return shapely.geometry.shape({'type': point_or_polygon, 'coordinates': tuple(new_coordinates)})
 
-
+    def make_json_compatible(self, struct: Any) -> Any:
```

Comment:
This method is redefined (overwritten) below.

Reply:
Apologies, this has to do with my errors during the merge